src/meta_tags.js

```diff
@@ -53,7 +53,10 @@ class MetaTags extends Component {
 
       this.lastChildStr = childStr;
 
-      let childNodes = Array.prototype.slice.call(this.temporaryElement.querySelector('.react-head-temp').children);
+      const tempHead = this.temporaryElement.querySelector('.react-head-temp');
+      let childNodes = tempHead === null
```

Comment:
Instead of this. You can just early return. if tempHead is null. Also can you add a comment on code about where this case happens. 
I assume this is happening when because of ReactDom.render being async method, before the render finishes if the MetaTag unmounts, the tempHead would not be available. In any other case it should be available.